stac_mjx/controller.py

```diff
@@ -25,11 +25,38 @@
 from tqdm import tqdm
 
 # Root position (3) + quaternion (7) in qpos
-_ROOT_QPOS_LB = -jp.inf * jp.ones(7)
-_ROOT_QPOS_UB = jp.inf * jp.ones(7)
+_ROOT_QPOS_LB = -jp.inf
```

Comment:
Seems like spatial can be +- inf, but angular (quat components should be +- 1.0.)